src/lib/route-builder.js

```diff
@@ -110,8 +110,8 @@ class RouteBuilder {
     const _nextHopBySourceAmount = this.routingTables.findBestHopForSourceAmount(
       sourceLedger, finalLedger, sourceTransfer.amount)
     if (!_nextHopBySourceAmount) throwAssetsNotTradedError()
-    // Round against ourselves since the quote rate was overestimated in our favor.
-    const nextHop = yield this._roundHop(_nextHopBySourceAmount, 'down', 'up')
+    // Round in favor of the connector.
+    const nextHop = yield this._roundHop(_nextHopBySourceAmount, 'up', 'down')
```

Comment:
Could you add some more comments to explain this more? Also, why is this changing from `'down', 'up'` to `'up', 'down'`? It would be to clarify more the thinking behind the logic for the next person who has to modify this code